vendor.sum

```diff
@@ -1,854 +0,0 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
- looks like the `vendor.sum` somehow got lost here 🙈 https://github.com/moby/moby/issues/51090